### PR TITLE
Add package directive to Makefile

### PR DESCRIPTION
**Pull request by 0pcom**
…d hypervisor

Did you run `make format && make check`?
No changes to the source were made outside the Makefile except adding systemd services. Possibly unnecessary for me to do that.

Partial fix for #387 

 Changes:	
- Add package directives to the Makefile for building amd64, arm64, and armhf .deb packages

How to test this PR:

Create packages for amd64, arm64, and armhf:
(use root so permissions are correct)
```
sudo make all-packages
```

install the correct package for your architecture (assuming you use amd64 debian):
```
sudo dpkg -i *amd64.deb
```

if you have configuration files in place at `/etc/skywire-viosor.json` or `/etc/skywire-hypervisor.json`, start the systemd service(s) of your choice and verify it's functionality

```
sudo systemctl start skywire-visor
sudo systemctl start skywire-hypervisor
```

### CODE REVIEW

**Comment by 0pcom** [comment]
Can someone please review this pull request?

**Review by jdknives — COMMENTED** [review]
I tested the package and it works well. Good job. This will not be merged yet.

File: static/skywire-visor.service
Comment (jdknives) [inline static/skywire-visor.service]: This will be changed to ` ExecStart=/usr/bin/skywire-visor -c /etc/skywire-visor.json` after the latest changes to `develop`.

File: Makefile
Comment (evanlinjin) [inline Makefile]: For the following targets, can we put the majority of the logic here in a separate script under the `/scripts` directory? Thank you

File: static/skywire-hypervisor.service
Comment (evanlinjin) [inline static/skywire-hypervisor.service]: Can we rename this directory (`/static`) to `/init`?

**Review by evanlinjin — CHANGES_REQUESTED** [review]
Good job. Please review my comments.

File: Makefile
Comment (0pcom) [reply Makefile]: Yes, it's preferable for this to all be in a script. I did it this way as a demonstration.

File: static/skywire-hypervisor.service
Comment (0pcom) [reply static/skywire-hypervisor.service]: I don't see why not. The structure of the repo is up to you and Rudi. I chose static because that is where the systemd services were in skybian.

File: static/skywire-visor.service
Comment (0pcom) [reply static/skywire-visor.service]: noted

Comment (0pcom) [reply static/skywire-visor.service]: fixed

File: Makefile
Comment (0pcom) [reply Makefile]: done

File: static/skywire-hypervisor.service
Comment (0pcom) [reply static/skywire-hypervisor.service]: done

**Comment by 0pcom** [comment]
Good to go, pending your approval.

File: Makefile
Comment (jdknives) [reply Makefile]: Would you mind moving the script into /scripts?

**Comment by jdknives** [comment]
@evanlinjin please review this again

File: Makefile
Comment (evanlinjin) [inline Makefile]: Is there a need to create a symbolic link before running the script?

File: scripts/dPKGBUILD.sh
Comment (evanlinjin) [inline scripts/dPKGBUILD.sh]: We should not be calling `make` here. This should be specified within the make target:
```
all-packages: install-deps-ui lint-ui build-ui
```
Or something.

**Review by evanlinjin — CHANGES_REQUESTED** [review]
Except for these minor issues, this works great! Good job.

File: Makefile
Comment (0pcom) [reply Makefile]: It's meant to be run from the skywire-mainnet directory, but otherwise no. I just wanted to make sure it executed in the same way as I had tested it when I moved the script into `scripts`.

File: scripts/dPKGBUILD.sh
Comment (0pcom) [reply scripts/dPKGBUILD.sh]: You're right, I just wanted it to work as a standalone script. Fixed

**Review by evanlinjin — APPROVED** [review]
This looks good. Good job!

**Comment by 0pcom** [comment]
@evanlinjin many thanks for your approval and support

@jdknives when can this be merged?

i would update the README.md about this but i dont want to delay the merge at this point.

**Comment by jdknives** [comment]
> @evanlinjin many thanks for your approval and support
> 
> @jdknives when can this be merged?
> 
> i would update the README.md about this but i dont want to delay the merge at this point.

Will merge this now. @0pcom good job.

**Pull request merged**
